sql/src/main/java/org/apache/druid/sql/calcite/schema/DruidSchema.java

```diff
@@ -168,6 +173,16 @@ public DruidSchema(
             removeSegment(segment);
             return ServerView.CallbackAction.CONTINUE;
           }
+
+          @Override
+          public ServerView.CallbackAction serverSegmentRemoved(
+              final DruidServerMetadata server,
+              final DataSegment segment
+          )
+          {
+            removeSegment(server, segment);
```

Comment:
nit leftover overloaded `removeSegment` method here seems confusing, maybe rename similar to event being renamed.

Reply:
renamed